data/pages-content/tickets.md

```diff
@@ -45,131 +32,107 @@ Each of the three ticket types is available in three payment tiers.
 We encourage and trust you to pick a fair ticket tier that fits your personal
 circumstance. The money you spend will be put straight back into the community
 to support our diversity initiatives,
-[financial aid programmes](/finaid) and
+[financial aid programmes](https://ep2023.europython.eu/finaid) and
 [other grants](https://www.europython-society.org/grants/)
 offered to the European Python community.
 
--   **Business Tickets**: for attendees whose company/business is paying for
-    them to attend, or if you use Python professionally. As someone with the
-    means to afford this tier you help us keep the conference affordable for
-    everyone. Thank you!
--   **Personal Tickets**: for those who enjoy Python as a hobbyist or use it as
-    a freelancer.
--   **Education Tickets**: for students, and teachers working in the education
-    sector (your main employment is at a school, college or university).
+- **Business Tickets**: for attendees whose company/business is paying for
+  them to attend, or if you use Python professionally. As someone with the
+  means to afford this tier you help us keep the conference affordable for
+  everyone. Thank you!
+- **Personal Tickets**: for those who enjoy Python as a hobbyist or use it as
+  a freelancer.
+- **Education Tickets**: for students, and teachers working in the education
+  sector (your main employment is at a school, college or university). We offer Education Tickets below costs.
```

Comment:
I'm not sure if I like the phrasing of `We offer Education Tickets below costs.`, Can we say something along the lines of `We subsidise Education Tickets to make it accessible to everyone.`

This also helps in us not revealing the ball park figure of cost per person at EP